production/helm/promtail/values.yaml

```diff
@@ -119,7 +118,7 @@ config:
     external_labels: {}
 
   server:
-    http_listen_port: 3101
+    http_listen_port: {{ .Values.port }}
```

Comment:
Have you tested this? I originally had it this way but it doesn't work when passed via tpl due to it being a number.

Reply:
it is a upstream bug, i have found another way to do it.